src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup.mdx

```diff
@@ -11,23 +11,25 @@ redirects:
   - /docs/integrations/open-source-telemetry-integrations/opentelemetry/set-opentelemetry-new-relic
   - /docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start-guide
   - /docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-architecture-recipes/
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start
 ---
 
-OpenTelemetry is a flexible toolkit that you can implement in a variety of ways. We recommend a basic four-step approach for setting up OpenTelemetry with New Relic. Here's an overview of the process, followed by details for each step.
+OpenTelemetry is a flexible toolkit that you can implement in a variety of ways. We recommend a basic five-step approach for setting up OpenTelemetry with New Relic. Here's an overview of the process, followed by details for each step.
 
 1. [Prerequisites](#prereqs)
 2. [Instrument your service with OpenTelemetry](#instrument)
-3. [Export your telemetry data to New Relic](#export)
-4. [View your data in the New Relic UI](#view-data)
+3. [Review configurations for exporting telemetry data to New Relic](#review-settings)
+4. [Complete the export configuration steps](#export)
+5. [View your data in the New Relic UI](#view-data)
 
-## Step 1. Prerequisites [#prereqs]
+## Step 1: Prerequisites [#prereqs]
 
 First things first:
 
 * If we don't already know you, sign up for a free [New Relic account](https://newrelic.com/signup).
```

Comment:
I don't know if this is approved, cool way to do it, but I find the 'know you' concept a bit awkward.

Reply:
@zuluecho9, I dropped the hipster approach for this: "If you haven't already done so, sign up for a free...."